core/app/modules/onboarding/assets/js/components/layout/header.scss

```diff
@@ -36,6 +36,20 @@
 		.eicon-pro-icon {
 			color: $brand-color-dark-pink;
 		}
+
+		&.onboarding-go-pro__btn {
+			background-color: $platform-accent;
+			color: $white;
+			padding: 4px 8px;
+			border-radius: 4px;
+			font-weight: 700;
+			font-size: 12px;
+			transition: 0.5s;
+
+			&:hover {
+			  	background-color: $platform-accent-hover;
```

Comment:
The Figma reference (global widgets go-pro button) uses this:

```SCSS
background: lighten($editor-accent, 10%)
```

Maybe use this instead of adding a new color, or use the same lighten function with the existing `$platform-accent` color

Reply:
How can I know that `lighten($editor-accent, 10%)` is the color that was intended in the Figma?
E.g, `background: lighten($editor-accent, 10%)` is `#d7306a` and in the figma they ask for `#B52652`

Reply:
mmm... now I see it is the same color... 
Yalla, resolved.